### ISSUE TITLE
about the nested device

### ISSUE THREAD

**Issue by webb-lu** [sitewhere/sitewhere-android#2]
hi, I'm testing the function about nesting device.
But when I sent the command to the child device, the parent device didn't receive the same command.

The parent device is a Gateway Device, and the child device is an Android Default device,
these both assign to the same asset, and I set the Android device fill in the COM Port1 slot (/default/serial/com1) in the composition .

the map of the composition like below:
```
Device Slots 
> Gateway Port 1 (/gw1) - "Gateway Device"
Default Bus (/default)
>Serial Ports (/default/serial)
>> COM Port 1 (/default/serial/com1) - "Android Device"
```

Do I misunderstand the usage of this function?
Thanks for help.

**Comment by webb-lu** [comment]
And I try this structure like below:
```
Device Slots 
> Gateway Port 1 (/gw1) - "Android Device"
```

And I use Restful API to get the device info, 
it shows the "Gateway Device" is the parent of the "Android Device"
but when I send the command to the android, the gateway still didn't get any command.

**Comment by derekadams** [comment]
I was able to replicate the issue so this appears to be a bug. A Jira issue has been created for the bug (https://sitewhere.atlassian.net/browse/SITEWHERE-130) and it should be resolved for the 1.0.1 release which will be available in the next couple of weeks. I'm going to close this report and future updates will be in Jira. Thanks for reporting the issue!
